Automattic/vip-cli
# Add Rollbar to VIP-CLI

## Description

This includes Rollbar to VIP-CLI.

By default, unhandled & uncaught rejections are both handled. But we can report other things to Rollbar if needed. Using `rollbar.error`, `rollbar.info`... will report it directly to Rollbar.

## Steps to Test

Outline the steps to test and verify the PR here.

Example:

1. Check out PR.
2. Run `npm install`
3. Raise an error somewhere
4. Execute the command raising the error
5. Check if it's showing in Rollbar
